### ISSUE TITLE
Feature - Sample dashboards

### ISSUE THREAD

**Issue by lhain08** [lhain08/FitbitDashboard#55]
Depends on issue #54.

Create some sample dashboards and allow users to select from some pre-made dashboards when creating new dashboards, if a sample is selected, the new dashboard will automatically be populated with some pre-set widgets.

This is a low priority issue and would best be put off until some other features like widget sizing is completed.
